src/Pages/taeheechoi/Main/Main.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './Main.scss';
+
+import Header from './Header.Header';
+import MainLeft from './MainLeft.MainLeft';
+import MainRight from './MainRight.MainRight';
+import { Redirect } from 'react-router-dom';
+
+class Main extends React.Component {
+    render() {
+        return (
+            <div>
+                <Header />
+                <div className="main">
```

Comment:
```suggestion
            <div className="Main">
                <Header />
                <div>
```
최상위 태그에 클래스명 컴포넌트 이름과 같게 해주세요.